app/components/preprint-form-authors.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
```

Comment:
Hmm. There are a few considerations here. So far, we've been using pods in ember-osf and osf.io apps, according to the thinking below:

- We should probably be consistent across projects to avoid fragmentation. Ember-osf, osf.io, and ember-share all use pod style at the moment, for reference.

- The "classic" structure puts related files in too many different places, making it hard to clean up (or to see everything required for the component to work). At a minimum, styles should be added to their own css file, not app.css. (I think it's reasonable to expect that we will refactor as requirements evolve; we should set a standard that minimizes unused files)
  - Dawn does this well. Some of our other PRs present, less so. https://github.com/CenterForOpenScience/ember-preprints/pull/20#discussion-diff-74318401R751

- Pods do have some major drawbacks; we had to do some hoop jumping to make ember-osf recognize style files outside the `styles/` folder. It's genuinely annoying, but a solvable problem.

- Pod-like behavior is the direction ember is headed, though not pods themselves. The official long-term picture is probably the unfinished [module unification](https://github.com/emberjs/rfcs/pull/143) RFC. They specifically call out the "titlebar problem" of confusing filenames. :)

-